config/olm/openshift/bundle-0.2.1.Dockerfile

```diff
@@ -18,6 +18,6 @@ LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
 # Copy files to locations specified by labels.
 COPY 0.2.1/manifests /manifests/
 COPY 0.2.1/metadata /metadata/
-LABEL com.redhat.openshift.versions="v4.5,v4.6,v4.7"
+LABEL com.redhat.openshift.versions="v4.5,v4.6"
```

Comment:
Already released version, we just keep the files for history purposes

Reply:
Ammended/Removed this change